### ISSUE TITLE
Exception occurred when creating MZContentProviderUpload for provider. (1004)

### ISSUE THREAD

**Issue by Panajev** [fastlane/fastlane#17330]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

```
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter Checksums = {81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa=ddffb3a901e33f5e45c2e30fc254bb40}
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter FileSizes = {81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa=131887246}
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter ErrorCode = 1004
DEBUG [2020-09-28 18:41:48.16]: [Transporter]: DBG-X:   parameter ErrorMessage = Exception occurred when creating MZContentProviderUpload for provider. (1004)
DEBUG [2020-09-28 18:41:48.16]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
DEBUG [2020-09-28 18:41:48.16]: [Transporter]: DBG-X:   parameter ProviderNameForToken = <...>
ERROR [2020-09-28 18:41:48.16]: [Transporter Error Output]: Exception occurred when creating MZContentProviderUpload for provider. (1004)

DEBUG [2020-09-28 18:41:48.16]: [Transporter]: DBG-X: The error code is: 1004
DEBUG [2020-09-28 18:41:48.16]: [Transporter]: INFO: Done performing authentication.
ERROR [2020-09-28 18:41:48.21]: Transporter transfer failed.
WARN [2020-09-28 18:41:48.21]: 
ERROR [2020-09-28 18:41:48.21]: Exception occurred when creating MZContentProviderUpload for provider. (1004)

```

##### Command executed

`bundle exec fastlane deliver -z 11.16.6 --ipa /<JOBS>/build/<APP>.ipa  --skip_screenshots --skip_metadata --verbose --api_key_path /<JOBS>/.../signing/786G3G9X94.json --precheck_include_in_app_purchases false`

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> INFO [2020-09-28 18:40:28.65]: Successfully loaded '/<JOBS_SRC>/<path_>/Deliverfile' 📄

+----------------+---------------------------------+
|       Detected Values from './Deliverfile'       |
+----------------+---------------------------------+
| app_identifier | <bundleID>     |
| username       | <usernameID> |
+----------------+---------------------------------+

INFO [2020-09-28 18:40:28.67]: Creating authorization token for App Store Connect API

+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------+
|                                                                     deliver 2.160.0 Summary                                                                     |
+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------+
| app_version                          | 11.16.6                                                                                                                  |
| ipa                                  | /<JOBS_SRC>/build/<APP>.ipa                           |
| skip_screenshots                     | true                                                                                                                     |
| skip_metadata                        | true                                                                                                                     |
| api_key_path                         | /<JOBS_SRC>/<ProjectAPP>/scripts/signing/786G3G9X94.json |
| precheck_include_in_app_purchases    | false                                                                                                                    |
| screenshots_path                     | ./screenshots                                                                                                            |
| metadata_path                        | ./metadata                                                                                                               |
| username                             | <usernameID>                                                                                          |
| app_identifier                       | <bundleID>                                                                                              |
| platform                             | ios                                                                                                                      |
| edit_live                            | false                                                                                                                    |
| use_live_version                     | false                                                                                                                    |
| skip_binary_upload                   | false                                                                                                                    |
| skip_app_version_update              | false                                                                                                                    |
| force                                | false                                                                                                                    |
| overwrite_screenshots                | false                                                                                                                    |
| submit_for_review                    | false                                                                                                                    |
| reject_if_possible                   | false                                                                                                                    |
| automatic_release                    | false                                                                                                                    |
| phased_release                       | false                                                                                                                    |
| reset_ratings                        | false                                                                                                                    |
| run_precheck_before_submit           | true                                                                                                                     |
| precheck_default_rule_level          | warn                                                                                                                     |
| ignore_language_directory_validation | false                                                                                                                    |
+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------+

INFO [2020-09-28 18:40:32.25]: Making sure the latest version on App Store Connect matches '11.16.6'...
INFO [2020-09-28 18:40:33.10]: '11.16.6' is the latest version on App Store Connect
INFO [2020-09-28 18:40:33.10]: Uploading binary to App Store Connect
INFO [2020-09-28 18:40:34.13]: Wrote XML data to '/tmp/723566418.itmsp'
INFO [2020-09-28 18:40:34.13]: Going to upload updated app to App Store Connect
INFO [2020-09-28 18:40:34.13]: This might take a few minutes. Please don't interrupt the script.
DEBUG [2020-09-28 18:40:34.13]: xcrun iTMSTransporter -m upload -jwt YourJWT -f /tmp/723566418.itmsp -k 100000 2>&1
DEBUG [2020-09-28 18:40:41.94]: [Transporter]: INFO: Configuring logging...
DEBUG [2020-09-28 18:40:41.95]: [Transporter]: INFO: Logging level set to eXtreme
DEBUG [2020-09-28 18:40:41.96]: [Transporter]: INFO: Transporter is searching for new software components.
DEBUG [2020-09-28 18:40:41.96]: [Transporter]: INFO: INFO: using cached repository.xml file.
DEBUG [2020-09-28 18:40:42.56]: [Transporter]: INFO: Update check complete.
DEBUG [2020-09-28 18:40:45.34]: [Transporter]: DEBUG: Attempting refresh of configuration data from https://contentdelivery.itunes.apple.com/transporter/Defaults.properties
DEBUG [2020-09-28 18:40:45.80]: [Transporter]: DEBUG: Configuration refresh successful.
DEBUG [2020-09-28 18:40:45.80]: [Transporter]: DEBUG: Saving configuration to local path: /Users/ci/Library/Caches/com.apple.amp.itmstransporter/Defaults.properties
DEBUG [2020-09-28 18:40:45.93]: [Transporter]: INFO: iTMSTransporter : iTunes Store Transporter [2.0.0]
DEBUG [2020-09-28 18:40:45.93]: [Transporter]: INFO: OS identifier: Mac OS X 10.15.7 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-1; jre=14.0.2+12-iTunesOpenJDK-1
DEBUG [2020-09-28 18:40:45.93]: [Transporter]: INFO: Memory: [JVM] 729M free, 1024M total, 2048M max [System] (Physical) 197M free, 12288M total (Swap) 1539M free, 2048M total
DEBUG [2020-09-28 18:40:45.93]: [Transporter]: DBG-X: Memory: [JVM] 729M free, 1024M total, 2048M max [System] (Physical) 196M free, 12288M total (Swap) 1539M free, 2048M total
DEBUG [2020-09-28 18:40:45.94]: [Transporter]: DBG-X: Using operation named: authenticateForSession
DEBUG [2020-09-28 18:40:45.94]: [Transporter]: DBG-X: Apple's web service operation input parameters:
DEBUG [2020-09-28 18:40:45.94]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
DEBUG [2020-09-28 18:40:45.94]: [Transporter]: DBG-X:   parameter BaseVersion = 2.0.0
DEBUG [2020-09-28 18:40:45.95]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.7 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-1; jre=14.0.2+12-iTunesOpenJDK-1
DEBUG [2020-09-28 18:40:45.95]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2020-09-28T18:40:45+01:00
DEBUG [2020-09-28 18:40:45.95]: [Transporter]: DBG-X:   parameter StatisticsPreviousCallDurationInSecs = 0.610046038
DEBUG [2020-09-28 18:40:45.95]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -f /tmp/723566418.itmsp -k 100000
DEBUG [2020-09-28 18:40:45.95]: [Transporter]: DBG-X:   parameter Version = 2.0.0
DEBUG [2020-09-28 18:40:45.95]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
DEBUG [2020-09-28 18:40:45.95]: [Transporter]: INFO: id = 20200928184045-623
DEBUG [2020-09-28 18:40:45.97]: [Transporter]: INFO: iTMSTransporter Correlation Key: 42f480c6-5967-4b7b-88b2-437b16393766-0001
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X: Apple's web service operation return value:
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X:   parameter SharedSecret = **hidden value**
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X:   parameter MultipartUploadsEnabled = true
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X:   parameter TxHeaders = {jenga=true}
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X:   parameter SessionExpiration = 2020-10-02T17:40:46.625Z
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X:   parameter SessionId = CJmqDBIQVfhCZErVQrKbNjQ9tIBNgQ==
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
DEBUG [2020-09-28 18:40:46.82]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
DEBUG [2020-09-28 18:40:46.84]: [Transporter]: INFO: Upload mode selected.
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X: Memory: [JVM] 990M free, 1024M total, 2048M max [System] (Physical) 152M free, 12288M total (Swap) 1539M free, 2048M total
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X: Using operation named: authenticateForTransportDiscovery
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X: Apple's web service operation input parameters:
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X:   parameter BaseVersion = 2.0.0
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.7 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-1; jre=14.0.2+12-iTunesOpenJDK-1
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2020-09-28T18:40:46+01:00
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -f /tmp/723566418.itmsp -k 100000
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X:   parameter Version = 2.0.0
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
DEBUG [2020-09-28 18:40:46.86]: [Transporter]: INFO: id = 20200928184046-706
DEBUG [2020-09-28 18:40:46.87]: [Transporter]: INFO: iTMSTransporter Correlation Key: 42f480c6-5967-4b7b-88b2-437b16393766-0001
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X: Apple's web service operation return value:
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X:   parameter PreferredTransports = [HTTP, Signiant, Aspera, DAV]
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X:   parameter TransportHosts.Signiant = [sgr604.apple.com:44001, sgr601.apple.com:44001, sgr603.apple.com:44001, sgr602.apple.com:44001]
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X:   parameter SessionExpiration = 2020-10-02T17:40:47.291Z
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X:   parameter TransportHosts.DAV = [itmsdav.apple.com:443]
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X:   parameter TransportHosts.Aspera = [vgr501.apple.com:33001, vgr502.apple.com:33001, vgr504.apple.com:33001, vgr503.apple.com:33001]
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
DEBUG [2020-09-28 18:40:47.64]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: INFO: Transporter's command line arguments are: -m upload -jwt **hidden value** -f /tmp/723566418.itmsp -k 100000
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: INFO: Examining the package at: /tmp/723566418.itmsp
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: INFO: Ensuring that package has well formed metadata file...
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: DEBUG: Getting case-sensitive metadata xml filename...
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: INFO: Gathering the list of valid files from the package ...
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: DEBUG:   metadata.xml will be verified by Apple's web service to determine if it is a valid file.
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: DEBUG:   81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa will be verified by Apple's web service to determine if it is a valid file.
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: INFO: Finished gathering the list of valid files from the package.
DEBUG [2020-09-28 18:40:47.65]: [Transporter]: DEBUG:   Case-sensitive metadata filename is: metadata.xml
DEBUG [2020-09-28 18:40:47.66]: [Transporter]: INFO: Performing authentication of package 723566418.itmsp ...
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X: Memory: [JVM] 975M free, 1024M total, 2048M max [System] (Physical) 155M free, 12288M total (Swap) 1539M free, 2048M total
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X: Deflated 484 bytes to 424
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X: Using operation named: validateMetadata
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X: Apple's web service operation input parameters:
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter BaseVersion = 2.0.0
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter Files = [metadata.xml, 81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa]
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter Metadata = <?xml version="1.0" encoding="UTF-8"?>
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter MetadataChecksum = 4421c127c8b8ddbdaf5fc49bf6b14a3d
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter MetadataCompressed = (suppressed)
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter MetadataInfo = {app_platform=ios, primary_bundle_identifier=, device_id=, bundle_identifier=, packageVersion=software5.4, apple_id=723566418, asset_types=[bundle], bundle_version=, bundle_short_version_string=}
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.7 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-1; jre=14.0.2+12-iTunesOpenJDK-1
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter PackageName = 723566418.itmsp
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter PackageSize = 131887730
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2020-09-28T18:40:47+01:00
DEBUG [2020-09-28 18:40:47.68]: [Transporter]: DBG-X:   parameter Throttle = 100000
DEBUG [2020-09-28 18:40:47.69]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -f /tmp/723566418.itmsp -k 100000
DEBUG [2020-09-28 18:40:47.69]: [Transporter]: DBG-X:   parameter Version = 2.0.0
DEBUG [2020-09-28 18:40:47.69]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
DEBUG [2020-09-28 18:40:47.69]: [Transporter]: INFO: id = 20200928184047-392
DEBUG [2020-09-28 18:40:47.69]: [Transporter]: INFO: iTMSTransporter Correlation Key: 42f480c6-5967-4b7b-88b2-437b16393766-0001
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X: Apple's web service operation return value:
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter SessionExpiration = 2020-10-02T17:40:47.810Z
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter MinimumFileSizeThresholdForCheckum = 104857600
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionFormat = binary
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionThresholds = {in-app-purchase-content={extensions=[pkg], size=100000000}, ccats={extensions=[pdf], size=100000000}, product-archive={extensions=[pkg], size=100000000}, developer-id-package={extensions=[zip, pkg, dmg], size=100000000}, bundle={extensions=[zip, ipa], size=100000000}, large-icon={extensions=[jpg, png, jpeg], size=100000000}}
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter Success = true
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter FilesToUpload = [metadata.xml, 81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa]
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter AssetsToDescribe = [{Role=source, Filename=81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa, ShouldDescribeAsset=true, UTI=com.apple.ipa}]
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter CPUToken = 949e63b3-82b6-45fe-88cf-b29480381081/1601314848512
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter Checksums = {81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa=ddffb3a901e33f5e45c2e30fc254bb40}
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter FileSizes = {81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa=131887246}
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
DEBUG [2020-09-28 18:40:48.62]: [Transporter]: DBG-X:   parameter ProviderNameForToken = <CompanyNameToken>
DEBUG [2020-09-28 18:40:48.63]: [Transporter]: INFO: The list of files requested for upload by Apple is: [metadata.xml, 81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa].  These are the files that will be uploaded.
DEBUG [2020-09-28 18:40:48.63]: [Transporter]: INFO: Starting media analysis of assets
DEBUG [2020-09-28 18:40:48.95]: [Transporter]: DEBUG: Handling asset: filename=81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa,uti=com.apple.ipa,role=source,describe=true
DEBUG [2020-09-28 18:40:48.95]: [Transporter]: DEBUG: Using container type tool name and version: iOS App Describer:1.26
DEBUG [2020-09-28 18:40:48.95]: [Transporter]: DEBUG: Using path: /tmp/723566418.itmsp/81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa
DEBUG [2020-09-28 18:41:24.51]: [Transporter]: DBG-X: description length:0
DEBUG [2020-09-28 18:41:24.51]: [Transporter]: DBG-X: Asset description: 
DEBUG [2020-09-28 18:41:24.51]: [Transporter]: DEBUG: Asset successfully described: com.apple.jingle.leghorn.software.AbstractSoftwareDescriber$1@55c57422
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X: Deflated 646 bytes to 412
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: INFO: Asset media analysis has completed
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X: Memory: [JVM] 944M free, 1024M total, 2048M max [System] (Physical) 683M free, 12288M total (Swap) 1539M free, 2048M total
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X: Using operation named: lookupTransportDiagnostic
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X: Apple's web service operation input parameters:
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X:   parameter BaseVersion = 2.0.0
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.7 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-1; jre=14.0.2+12-iTunesOpenJDK-1
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2020-09-28T18:41:24+01:00
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -f /tmp/723566418.itmsp -k 100000
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X:   parameter Version = 2.0.0
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: INFO: id = 20200928184124-147
DEBUG [2020-09-28 18:41:24.63]: [Transporter]: INFO: iTMSTransporter Correlation Key: 42f480c6-5967-4b7b-88b2-437b16393766-0001
DEBUG [2020-09-28 18:41:25.34]: [Transporter]: DBG-X: Apple's web service operation return value:
DEBUG [2020-09-28 18:41:25.34]: [Transporter]: DBG-X:   parameter AsperaLoggingLevel = OFF
DEBUG [2020-09-28 18:41:25.34]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
DEBUG [2020-09-28 18:41:25.34]: [Transporter]: DBG-X:   parameter SigniantLoggingLevel = OFF
DEBUG [2020-09-28 18:41:25.34]: [Transporter]: DBG-X:   parameter SessionExpiration = 2020-10-02T17:41:24.755Z
DEBUG [2020-09-28 18:41:25.34]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
DEBUG [2020-09-28 18:41:25.35]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
DEBUG [2020-09-28 18:41:25.38]: [Transporter]: DBG-X: Using operation named: createTestReservation
DEBUG [2020-09-28 18:41:25.38]: [Transporter]: DBG-X: Apple's web service operation input parameters:
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: DBG-X:   parameter BaseVersion = 2.0.0
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: DBG-X:   parameter fileDescriptions = ( one FileDescription )
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: DBG-X:   parameter NewPackageName = http-transport-diagnostic-20200928184125-BST-8736.itmsp
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.7 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-1; jre=14.0.2+12-iTunesOpenJDK-1
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -f /tmp/723566418.itmsp -k 100000
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: DBG-X:   parameter Version = 2.0.0
DEBUG [2020-09-28 18:41:25.39]: [Transporter]: INFO: id = 20200928184125-438
DEBUG [2020-09-28 18:41:25.53]: [Transporter]: INFO: iTMSTransporter Correlation Key: 977ff6b9-43e9-4c1d-95b3-fb88ce84c073-0001
DEBUG [2020-09-28 18:41:26.10]: [Transporter]: DBG-X: Apple's web service operation return value:
DEBUG [2020-09-28 18:41:26.10]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
DEBUG [2020-09-28 18:41:26.10]: [Transporter]: DBG-X:   parameter Reservations = ( one Reservation )
DEBUG [2020-09-28 18:41:26.10]: [Transporter]: DBG-X:   parameter StreamingSettings = {MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false, LogStreamingEnabled=true}
DEBUG [2020-09-28 18:41:26.10]: [Transporter]: DBG-X:   parameter Success = true
DEBUG [2020-09-28 18:41:26.10]: [Transporter]: INFO: Setting transport log file: /var/folders/34/yvy5l0h52ln7rqmrnh_b361m0000gq/T/http-transport-diagnostic-20200928184125-BST-8736.itmsp7107278411265864036.log
DEBUG [2020-09-28 18:41:26.10]: [Transporter]: INFO: Starting upload for packageName: http-transport-diagnostic-20200928184125-BST-8736.itmsp
DEBUG [2020-09-28 18:41:26.11]: [Transporter]: INFO: Thread Pool Configuration: {processors=8, perCoreThreadMultiplier=1.65, coreThreads=6, maxThreads=14, minConnections=6, maxConnections=13, rateLimit=100000 KBs}
DEBUG [2020-09-28 18:41:26.12]: [Transporter]: INFO: Reservation CPUSource124/v4/fc/bc/ab/fcbcab45-810a-1391-9f6c-37124db6493a/http-transport-diagnostic-16881018452831361493.txt created for file: (http-transport-diagnostic-16881018452831361493.txt), part count:(1), file size: (200)
DEBUG [2020-09-28 18:41:26.12]: [Transporter]: INFO: Computing total size of files to upload ...
DEBUG [2020-09-28 18:41:26.12]: [Transporter]: DBG-X:   adding file for % complete logging: http-transport-diagnostic-16881018452831361493.txt , size: 200
DEBUG [2020-09-28 18:41:26.12]: [Transporter]: DEBUG: Setting total size of files for upload to: 200
DEBUG [2020-09-28 18:41:26.12]: [Transporter]: INFO: Done computing total size of files to upload.
DEBUG [2020-09-28 18:41:26.13]: [Transporter]: INFO: Starting upload for reservation: (CPUSource124/v4/fc/bc/ab/fcbcab45-810a-1391-9f6c-37124db6493a/http-transport-diagnostic-16881018452831361493.txt) file: (http-transport-diagnostic-16881018452831361493.txt/1) size: 200
DEBUG [2020-09-28 18:41:26.71]: [Transporter]: INFO:      File: http-transport-diagnostic-16881018452831361493.txt 200/200, 100.00% completed
DEBUG [2020-09-28 18:41:26.71]: [Transporter]: INFO: Package upload progress: 100.00% completed
DEBUG [2020-09-28 18:41:26.94]: [Transporter]: INFO: Finished part upload for: (http-transport-diagnostic-16881018452831361493.txt/1) 858.37 B/s in 0 seconds
DEBUG [2020-09-28 18:41:26.94]: [Transporter]: INFO: Finished upload for reservation: (CPUSource124/v4/fc/bc/ab/fcbcab45-810a-1391-9f6c-37124db6493a/http-transport-diagnostic-16881018452831361493.txt) file: (http-transport-diagnostic-16881018452831361493.txt)
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: INFO: Waiting on reservation commit(s) to finish.
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X: Using operation named: commitTestReservation
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X: Apple's web service operation input parameters:
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X:   parameter BaseVersion = 2.0.0
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X:   parameter NewPackageName = http-transport-diagnostic-20200928184125-BST-8736.itmsp
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.7 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-1; jre=14.0.2+12-iTunesOpenJDK-1
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X:   parameter reservations = [CPUSource124/v4/fc/bc/ab/fcbcab45-810a-1391-9f6c-37124db6493a/http-transport-diagnostic-16881018452831361493.txt]
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -f /tmp/723566418.itmsp -k 100000
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: DBG-X:   parameter Version = 2.0.0
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: INFO: id = 20200928184126-300
DEBUG [2020-09-28 18:41:26.95]: [Transporter]: INFO: iTMSTransporter Correlation Key: 977ff6b9-43e9-4c1d-95b3-fb88ce84c073-0001
DEBUG [2020-09-28 18:41:27.14]: [Transporter]: DBG-X: Apple's web service operation return value:
DEBUG [2020-09-28 18:41:27.14]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
DEBUG [2020-09-28 18:41:27.14]: [Transporter]: DBG-X:   parameter StreamingSettings = {MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false, LogStreamingEnabled=true}
DEBUG [2020-09-28 18:41:27.14]: [Transporter]: DBG-X:   parameter Success = true
DEBUG [2020-09-28 18:41:27.14]: [Transporter]: INFO: Reservation committed: (CPUSource124/v4/fc/bc/ab/fcbcab45-810a-1391-9f6c-37124db6493a/http-transport-diagnostic-16881018452831361493.txt) file: (http-transport-diagnostic-16881018452831361493.txt)
DEBUG [2020-09-28 18:41:27.14]: [Transporter]: INFO: All reservation commit(s) finished.
DEBUG [2020-09-28 18:41:27.14]: [Transporter]: INFO: 195.69 B/s in 1 second
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DEBUG: Using token file path /Users/ci/Library/Caches/com.apple.amp.itmstransporter/UploadTokens/GL04712M.local_<CompanyNameToken>_723566418.itmsp.token
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DEBUG: Attempting exclusive file lock on token file
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DEBUG: Set token: 949e63b3-82b6-45fe-88cf-b29480381081/1601314848512
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X: Memory: [JVM] 925M free, 1024M total, 2048M max [System] (Physical) 665M free, 12288M total (Swap) 1539M free, 2048M total
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X: Deflated 484 bytes to 424
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X: Using operation named: validateAssets
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X: Apple's web service operation input parameters:
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter BaseVersion = 2.0.0
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter CPUToken = 949e63b3-82b6-45fe-88cf-b29480381081/1601314848512
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter Files = [metadata.xml, 81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa]
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter MetadataChecksum = 4421c127c8b8ddbdaf5fc49bf6b14a3d
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter MetadataCompressed = (suppressed)
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter MetadataInfo = {app_platform=ios, primary_bundle_identifier=, device_id=, bundle_identifier=, packageVersion=software5.4, apple_id=723566418, asset_types=[bundle], bundle_version=, bundle_short_version_string=}
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.7 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-1; jre=14.0.2+12-iTunesOpenJDK-1
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter PackageName = 723566418.itmsp
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter PackageSize = 131887730
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter RequestUUID = 2e095aac-5b5f-47af-aaa5-15246038286c
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2020-09-28T18:41:27+01:00
DEBUG [2020-09-28 18:41:27.15]: [Transporter]: DBG-X:   parameter StreamingInfoList = [file: 8183d2a2-dfd5-445f-a56b-469b644177c22251265037063199517.swinfo.plist.gz, content-length: 664891, content-type: application/binary, MD5: 4b49b8779d15da02158cdfc3dd958dd3]
DEBUG [2020-09-28 18:41:27.16]: [Transporter]: DBG-X:   parameter Throttle = 100000
DEBUG [2020-09-28 18:41:27.16]: [Transporter]: DBG-X:   parameter Transport = HTTP
DEBUG [2020-09-28 18:41:27.16]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -f /tmp/723566418.itmsp -k 100000
DEBUG [2020-09-28 18:41:27.16]: [Transporter]: DBG-X:   parameter Version = 2.0.0
DEBUG [2020-09-28 18:41:27.16]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
DEBUG [2020-09-28 18:41:27.16]: [Transporter]: INFO: id = 20200928184127-067
DEBUG [2020-09-28 18:41:27.16]: [Transporter]: INFO: iTMSTransporter Correlation Key: 42f480c6-5967-4b7b-88b2-437b16393766-0001
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X: Apple's web service operation return value:
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter MinimumFileSizeThresholdForCheckum = 104857600
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionFormat = binary
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter RestartClient = false
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionThresholds = {in-app-purchase-content={extensions=[pkg], size=100000000}, ccats={extensions=[pdf], size=100000000}, product-archive={extensions=[pkg], size=100000000}, developer-id-package={extensions=[zip, pkg, dmg], size=100000000}, bundle={extensions=[zip, ipa], size=100000000}, large-icon={extensions=[jpg, png, jpeg], size=100000000}}
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter Success = false
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter Errors = [Exception occurred when creating MZContentProviderUpload for provider. (1004)]
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter FilesToUpload = [metadata.xml, 81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa]
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter AssetsToDescribe = [{Role=source, Filename=81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa, ShouldDescribeAsset=true, UTI=com.apple.ipa}]
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter CPUToken = 949e63b3-82b6-45fe-88cf-b29480381081/1601314848512
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter Checksums = {81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa=ddffb3a901e33f5e45c2e30fc254bb40}
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter FileSizes = {81174b3b6c5fe61afcc0c1fdfd84c10c3b9193a1e1eb09f68815e39daba5b809.ipa=131887246}
DEBUG [2020-09-28 18:41:48.15]: [Transporter]: DBG-X:   parameter ErrorCode = 1004
DEBUG [2020-09-28 18:41:48.16]: [Transporter]: DBG-X:   parameter ErrorMessage = Exception occurred when creating MZContentProviderUpload for provider. (1004)
DEBUG [2020-09-28 18:41:48.16]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
DEBUG [2020-09-28 18:41:48.16]: [Transporter]: DBG-X:   parameter ProviderNameForToken = <CompanyNameToken>
ERROR [2020-09-28 18:41:48.16]: [Transporter Error Output]: Exception occurred when creating MZContentProviderUpload for provider. (1004)

DEBUG [2020-09-28 18:41:48.16]: [Transporter]: DBG-X: The error code is: 1004
DEBUG [2020-09-28 18:41:48.16]: [Transporter]: INFO: Done performing authentication.
ERROR [2020-09-28 18:41:48.21]: Transporter transfer failed.
WARN [2020-09-28 18:41:48.21]: 
ERROR [2020-09-28 18:41:48.21]: Exception occurred when creating MZContentProviderUpload for provider. (1004)

WARN [2020-09-28 18:41:48.32]: [iTMSTransporter] [2020-09-28 18:41:48 BST] <main> DBG-X:   parameter ErrorMessage = Exception occurred when creating MZContentProviderUpload for provider. (1004)

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] [2020-09-28 18:41:48 BST] <main> DBG-X:   parameter ShouldUseRESTAPIs = false

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] [2020-09-28 18:41:48 BST] <main> DBG-X:   parameter ProviderNameForToken = <CompanyNameToken>

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] [2020-09-28 18:41:48 BST] <main> ERROR: Exception occurred when creating MZContentProviderUpload for provider. (1004)

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] [2020-09-28 18:41:48 BST] <main> DBG-X: The error code is: 1004

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] [2020-09-28 18:41:48 BST] <main>  INFO: Done performing authentication.

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] 

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] 

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] 

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] Package Summary:

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter]  

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] 1 package(s) were not uploaded because they had problems:

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] 	/tmp/723566418.itmsp - Error Messages:

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] 		Exception occurred when creating MZContentProviderUpload for provider. (1004)

WARN [2020-09-28 18:41:48.33]: [iTMSTransporter] [2020-09-28 18:41:48 BST] <main> DBG-X: Returning 1

INFO [2020-09-28 18:41:48.33]: iTunes Transporter output above ^
ERROR [2020-09-28 18:41:48.33]: Exception occurred when creating MZContentProviderUpload for provider. (1004)
Return status of iTunes Transporter was 1: Exception occurred when creating MZContentProviderUpload for provider. (1004)
The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.

Looking for related GitHub issues on fastlane/fastlane...
Search query: Could not upload binary to App Store Connect. Check out the error above

URL: https://api.github.com/search/issues?q=Could%20not%20upload%20binary%20to%20App%20Store%20Connect.%20Check%20out%20the%20error%20above+repo:fastlane/fastlane
➡️  [pilot] Uploading an IPA to TestFlight or app-store fails under Linux
    https://github.com/fastlane/fastlane/issues/16996 [open] 2 💬
    2 weeks ago

➡️  DELIVER (2.150.0.rc3/4) always ask attribute whatsNew
    https://github.com/fastlane/fastlane/issues/16700 [open] 44 💬
    a week ago

➡️  [fastlane_core, pilot] Improved transporter logging to pilot
    https://github.com/fastlane/fastlane/pull/12259 [open] 24 💬
    26 May 2020

and 118 more at: https://github.com/fastlane/fastlane/search?q=Could%20not%20upload%20binary%20to%20App%20Store%20Connect.%20Check%20out%20the%20error%20above&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.
bundler: failed to load command: fastlane (/<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/bin/fastlane)
FastlaneCore::Interface::FastlaneError: [!] Could not upload binary to App Store Connect. Check out the error above
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/fastlane_core/lib/fastlane_core/ui/interface.rb:141:in `user_error!'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/deliver/lib/deliver/runner.rb:182:in `upload_binary'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/deliver/lib/deliver/runner.rb:58:in `run'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/deliver/lib/deliver/commands_generator.rb:81:in `block (2 levels) in run'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/deliver/lib/deliver/commands_generator.rb:186:in `run'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/deliver/lib/deliver/commands_generator.rb:17:in `start'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/fastlane/lib/fastlane/cli_tools_distributor.rb:111:in `take_off'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/fastlane-2.160.0/bin/fastlane:23:in `<top (required)>'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/bin/fastlane:23:in `load'
  /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/bin/fastlane:23:in `<top (required)>'
Build step 'Execute shell' marked build as failure
Collecting metadata...
Metadata collection done.
sent [/<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/extensions/x86_64-darwin-19/2.6.0/ffi-1.13.1/mkmf.log, /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/extensions/x86_64-darwin-19/2.6.0/json-2.3.1/mkmf.log, /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/extensions/x86_64-darwin-19/2.6.0/unf_ext-0.0.7.7/mkmf.log, /<JOBS_SRC>/<path_>/vendor/bundle/ruby/2.6.0/gems/xcpretty-0.3.0/features/fixtures/xcodebuild.log, /<JOBS_SRC>/build/Packaging.log] to splunk in 46 events
[Checks API] No suitable checks publisher found.
Finished: FAILURE </pre>
</details>

### Environment

<details>
  <pre>[19:03:00]: Generating fastlane environment output, this might take a few seconds...
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                                               |
| --------------------------- | ----------------------------------------------------------------------------------- |
| OS                          | 10.15.7                                                                             |
| Ruby                        | 2.6.6                                                                               |
| Bundler?                    | true                                                                                |
| Git                         | git version 2.28.0                                                                  |
| Installation Source         | ~/ttemp/src___/vendor/bundle/ruby/2.6.0/bin/fastlane |
| Host                        | Mac OS X 10.15.7 (19H2)                                                             |
| Ruby Lib Dir                | ~/.rbenv/versions/2.6.6/lib                                                         |
| OpenSSL Version             | OpenSSL 1.1.1d  10 Sep 2019                                                         |
| Is contained                | false                                                                               |
| Is homebrew                 | false                                                                               |
| Is installed via Fabric.app | false                                                                               |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                                         |
| Xcode Version               | 12.0.1                                                                              |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.160.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| bundler                   | 2.1.4        |
| rake                      | 13.0.1       |
| CFPropertyList            | 3.0.2        |
| concurrent-ruby           | 1.1.7        |
| i18n                      | 1.8.5        |
| minitest                  | 5.14.2       |
| thread_safe               | 0.3.6        |
| tzinfo                    | 1.2.7        |
| activesupport             | 5.2.4.4      |
| public_suffix             | 4.0.6        |
| addressable               | 2.7.0        |
| httpclient                | 2.8.3        |
| json                      | 2.3.1        |
| algoliasearch             | 1.27.4       |
| atomos                    | 0.1.3        |
| aws-eventstream           | 1.1.0        |
| aws-partitions            | 1.376.0      |
| aws-sigv4                 | 1.2.2        |
| jmespath                  | 1.4.0        |
| aws-sdk-core              | 3.108.0      |
| aws-sdk-kms               | 1.38.0       |
| aws-sdk-s3                | 1.81.1       |
| babosa                    | 1.0.3        |
| claide                    | 1.0.3        |
| colored2                  | 3.1.2        |
| cork                      | 0.3.0        |
| nap                       | 1.1.0        |
| open4                     | 1.3.4        |
| claide-plugins            | 0.9.2        |
| fuzzy_match               | 2.0.4        |
| netrc                     | 0.11.0       |
| ffi                       | 1.13.1       |
| ethon                     | 0.12.0       |
| typhoeus                  | 1.4.0        |
| cocoapods-core            | 1.10.0.rc.1  |
| cocoapods-deintegrate     | 1.0.4        |
| cocoapods-downloader      | 1.4.0        |
| cocoapods-plugins         | 1.0.0        |
| cocoapods-search          | 1.0.0        |
| cocoapods-trunk           | 1.5.0        |
| cocoapods-try             | 1.2.0        |
| escape                    | 0.0.4        |
| fourflusher               | 2.3.1        |
| gh_inspector              | 1.1.3        |
| molinillo                 | 0.6.6        |
| ruby-macho                | 1.4.0        |
| nanaimo                   | 0.3.0        |
| xcodeproj                 | 1.18.0       |
| cocoapods                 | 1.10.0.rc.1  |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| multipart-post            | 2.0.0        |
| faraday                   | 1.0.1        |
| faraday-http-cache        | 2.2.0        |
| rchardet                  | 1.8.0        |
| git                       | 1.7.0        |
| rexml                     | 3.2.4        |
| kramdown                  | 2.3.0        |
| kramdown-parser-gfm       | 1.1.0        |
| no_proxy_fix              | 0.1.2        |
| sawyer                    | 0.8.2        |
| octokit                   | 4.18.0       |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| danger                    | 8.0.5        |
| ox                        | 2.13.4       |
| danger-junit              | 1.0.2        |
| danger-plugin-api         | 1.0.0        |
| danger-swiftformat        | 0.7.0        |
| thor                      | 0.20.3       |
| danger-swiftlint          | 0.24.4       |
| dotenv                    | 2.7.6        |
| emoji_regex               | 3.0.0        |
| excon                     | 0.76.0       |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.7        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.2.0        |
| jwt                       | 2.2.2        |
| memoist                   | 0.16.2       |
| multi_json                | 1.15.0       |
| os                        | 1.1.1        |
| signet                    | 0.14.0       |
| googleauth                | 0.13.1       |
| mini_mime                 | 1.0.2        |
| declarative               | 0.0.20       |
| declarative-option        | 0.1.0        |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| google-api-client         | 0.38.0       |
| digest-crc                | 0.6.1        |
| google-cloud-env          | 1.3.3        |
| google-cloud-errors       | 1.0.1        |
| google-cloud-core         | 1.5.0        |
| google-cloud-storage      | 1.29.0       |
| mini_magick               | 4.10.1       |
| plist                     | 3.5.0        |
| rubyzip                   | 2.3.0        |
| security                  | 0.1.3        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| slack-notifier            | 2.3.2        |
| terminal-notifier         | 2.0.0        |
| tty-screen                | 0.8.1        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| word_wrap                 | 1.0.0        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| xcpretty-travis-formatter | 1.0.0        |
| xcresult                  | 0.2.1        |
| xcov                      | 1.7.5        |
| danger-xcov               | 0.5.0        |
</details>

*generated on:* **2020-09-28**
</details>
[19:03:01]: Take notice that this output may contain sensitive information, or simply information that you don't want to make public.</pre>
</details>

**Comment by joshdholtz** [comment]
@Panajev I believe this is temporary issue. If you could try uploading again and/or use a different provider (👇) that would be great!

```sh
# Available options are: DAV, Aspera, Signiant
DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS=-t Aspera
```

**Comment by Panajev** [comment]
@joshdholtz  no problem, it happened to two different builds of two different apps. Might provide that parameter as a variable to the job later 👍

**Comment by banane** [comment]
This is occurring for me, as well

**Comment by joshdholtz** [comment]
I believe this is an issue on the upload server’s side so keep trying again after a little bit and it _should_ fix itself 🤞
